includes/wcdn-template-functions.php

```diff
@@ -356,17 +356,18 @@ function wcdn_get_order_info( $order, $type = '' ) {
 	$fields                = array();
 	$create_invoice_number = get_option( 'wcdn_create_invoice_number' );
 	$data                  = get_option( 'wcdn_' . $type . '_customization' );
-	$template              = $data['template_setting']['template_setting_template'];
+	if ( isset( $data['template_setting'] ) && is_array( $data['template_setting'] ) && isset( $data['template_setting']['template_setting_template'] ) ) {
+		$template = $data['template_setting']['template_setting_template'];
```

Comment:
Give a default value to $template if $data['template_setting']['template_setting_template'] is not set.